twake/backend/node/src/core/platform/services/storage/connectors/S3/service.ts

```diff
@@ -0,0 +1,23 @@
+import { Readable } from "stream";
+import { StorageConnectorAPI } from "../../provider";
+import * as Minio from "minio";
+
+export type S3Configuration = Minio.ClientOptions & { bucket: string };
+
+export default class S3ConnectorService implements StorageConnectorAPI {
+  client: Minio.Client;
+  minioConfiguration: S3Configuration;
+
+  constructor(S3Configuration: S3Configuration) {
+    this.client = new Minio.Client({ ...S3Configuration });
+    this.minioConfiguration = S3Configuration;
+  }
+
+  write(path: string, stream: Readable) {
```

Comment:
Even if `void` or `Promise<void>`, it will be nice to have the ouput type. Same for the local implementation